.circleci/config.yml

```
@@ -7,9 +7,6 @@ parameters:
   image_suffix:
     type: string
     default: '-v2021_10_27'
```

```suggestion
    default: '-dev-1e3a9bd'
```

We need to use the updated images here.

Currently we have a bug that is about to be fixed https://github.com/citusdata/the-process/pull/73

Once that lands, we can start using the suffixes without the `-dev` part